coralproject/talk
# I18n ro translation

## What does this PR do?

Adds partial Romanian translations to the project.

## How do I test this PR?

You switch it to Romanian language and check to see if the translatable strings are translated.
